core/input/input_event.cpp

```diff
@@ -1348,98 +1348,184 @@ void InputEventGesture::set_position(const Vector2 &p_pos) {
 	pos = p_pos;
 }
 
+Vector2 InputEventGesture::get_position() const {
+	return pos;
+}
+
+void InputEventGesture::set_touches(const int p_touches) {
+	touches = p_touches;
+}
+
+int InputEventGesture::get_touches() const {
+	return touches;
+}
+
 void InputEventGesture::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_position", "position"), &InputEventGesture::set_position);
 	ClassDB::bind_method(D_METHOD("get_position"), &InputEventGesture::get_position);
+	ClassDB::bind_method(D_METHOD("set_touches", "touches"), &InputEventGesture::set_touches);
+	ClassDB::bind_method(D_METHOD("get_touches"), &InputEventGesture::get_touches);
 
 	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "position"), "set_position", "get_position");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "touches"), "set_touches", "get_touches");
 }
 
-Vector2 InputEventGesture::get_position() const {
-	return pos;
+InputEventGesture::InputEventGesture() {
+	touches = 0;
+	pos = Vector2(0, 0);
 }
```

Comment:
`touches` can be initialized in the header, and then you can remove the constructor (`pos` doesn't need to be initialized manually, `Vector2` default initializes to `0, 0`).